core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -114,12 +114,13 @@ impl SchedulerController {
     ) -> Result<(), SchedulerError> {
         match decision {
             BufferedPacketsDecision::Consume(bank_start) => {
-                let (scheduling_summary, schedule_time_us) =
-                    measure_us!(self
-                        .scheduler
-                        .schedule(&mut self.container, |txs, results| {
-                            Self::pre_scheduling_filter(txs, results, &bank_start.working_bank)
-                        })?);
+                let (scheduling_summary, schedule_time_us) = measure_us!(self.scheduler.schedule(
+                    &mut self.container,
+                    |txs, results| {
+                        Self::pre_graph_filter(txs, results, &bank_start.working_bank)
+                    },
+                    |_| true // no pre-lock filter for now
```

Comment:
So this will eventually have logic in it, but for now just setting it as a dummy.
Plan is to introduce earlier fee-payer validation here

Reply:
curious what fee-payer validation going to look like? Leader's local cache that is recalibrated every slot(s)?

Reply:
Upcoming PR to just check fee_payer key against a bloom filter containing recently invalid fee payers.
**might** test out also just loading the account and checking it here as well; but will need the fee meta for that 😉